shanoir-ng-front/src/app/shared/components/entity/entity.abstract.service.ts

```diff
@@ -84,7 +86,7 @@ export abstract class EntityService<T extends Entity> {
                         }
                         let warn = 'The ' + name + (entity['name'] ? ' ' + entity['name'] : '') + ' with id ' + entity.id + ' is linked to other entities, it was not deleted.';
                         if((reason.error && reason.error.code == 422)
-                            || reason.status == 422){
+                            || reason.status == 422 || entity instanceof Manufacturer || entity instanceof ManufacturerModel){
```

Comment:
I'm not fan of putting this in a generic method, and I don't really see the purpose, I thought it was not possible to delete one anymore ?